nginx.conf

```diff
@@ -78,7 +78,15 @@ server {
     # checks for static file, if not found proxy to app
     try_files $uri @django;
   }
-
+  location /admin/ {
```

Comment:
Should we change it to `/admin/links`? @jsnshrmn, what do you think?

Reply:
I think the main thing is to not have this at `/admin/login`; all of the other paths should be fine